CMakeLists.txt

```diff
@@ -259,7 +260,10 @@ if (${CMAKE_SYSTEM_NAME} MATCHES "Linux")
 	install(FILES desktop/xournalpp.thumbnailer
 			DESTINATION ${CMAKE_INSTALL_PREFIX}/share/thumbnailers)
 
-	install(CODE "execute_process(COMMAND ${CMAKE_CURRENT_BINARY_DIR}/cmake/postinst configure)")
+	# TODO: Only package post install scripts with distro-specific packages
```

Comment:
Could be achieved with `CPACK_GENERATOR` if CPack is only ever used when a generator is specified explicitly (not sure how to handle the other case, we just use it that way).
See https://cmake.org/cmake/help/v3.0/module/CPack.html.

Reply:
`make install` should definitely include this command. Can we rewrite `postinst.in` to be distro-specific?

Reply:
Should `make install` automatically run this for *every* installation? Doesn't seem so. It shall be included in the final package, I guess.

AppImage creation uses `make install` as well, and it doesn't need the script at all. A tarball doesn't need it either I guess. It's only useful to native packages.

For AppImage and tarball, the script is therefore bloat. But that said, just packaging it doesn't harm anyone. Those few extra bytes don't hurt IMO.

In [AppImageLauncher](https://github.com/TheAssassin/AppImageLauncher/) I solved the issue by introducing [an extra variable](https://github.com/TheAssassin/AppImageLauncher/blob/master/resources/CMakeLists.txt#L20), so it's the duty of the pipeline job or packager to switch that on if the scripts are needed.